docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: astropy
+
+.. _environment_variables:
+
+*********************
+Environment variables
+*********************
+
+
+Runtime
+-------
+
+.. glossary::
+
+    ``XDG_CONFIG_HOME``
+        This environment variables control where configuration files are read
+        and written. Astropy will look for configuration files in
+        ``$XDG_CONFIG_HOME/astropy``.
+        See :ref:`astropy_config` for how to programmatically set or get the
+        location of the corresponding directory at runtime.
+
+    ``XDG_CACHE_HOME``
+        These environment variables control where data files are cached.
+        Astropy will cache files in ``$XDG_CACHE_HOME/astropy``.
+        See :ref:`utils-data` for how to programmatically set or get the
+        location of the corresponding directory at runtime.
+
+
+.. note::
+    ``XDG_CONFIG_HOME`` and ``XDG_CACHE_HOME`` come from a Linux-centric
+    specification
+    (see `here <https://wiki.archlinux.org/index.php/XDG_Base_Directory_support>`_
```

Comment:
Why is the link pointing to ArchWiki instead of the actual specification?

Reply:
Your guess is as good as mine.
my two cents: I think it's nice to point to a summary instead, especially when the actual specification is one obvious click away.

Reply:
It is always better to refer to the original source. Besides, it's a short specification.

Reply:
I feel it's out of scope to change it here. Can we please postpone this ?